.env

```diff
@@ -2,11 +2,13 @@
 # SPDX-License-Identifier: GPL-2.0-only
 # Copyright (c) 2022 Intel Corporation.
 
-lkvs_root=$(cd "$(dirname "$0")" && pwd)
+lkvs_root=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
 
 if [[ -z "$path_exported" ]]; then
-  NEW_PATH="${PATH}":$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")
+  NEW_PATH="${PATH}":${lkvs_root}:$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")
   NEW_PATH=${NEW_PATH%:}
```

Comment:
Please remove this line!  It's useless!

Because xxx:    ":" in the end.

it's for next path, if some one forget to add the PAH before  ":"
So it's reasonable.

If you add this line , it will cause some issue if some script add new path without ":"  in head!
Thanks!

Reply:
> Please remove this line! It's useless and cause issue.
> 
> Because xxx: ":" in the end.
> 
> it's for next path, if some one forget to add the PAH before ":" So it's reasonable.
> 
> If you add this line , it will cause some issue if some script add new path without ":" in head! Thanks!

Got your point. In my opinion, as there's no : at the end of $PATH by default (system $PATH), we should keep the convention. If someone need to append path to $PATH, they should assume there's no : at the end of $PATH, or they should check it by themself.

Reply:
It still seems unnecessary step due to useless.
Anyway,   we could keep it also.